RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -24,28 +24,35 @@ public static class WebViewPageExtensions
 
         private static readonly Dictionary<string, ViewPlaceholder> _views = new Dictionary<string, ViewPlaceholder>();
 
-        public static IViewEngine ViewEngine
+        private static StringWriter _writer;
+
+        public static DummyViewEngine ViewEngine
         {
             get { return _viewEngine; }
         }
 
+        public static StringWriter Writer
```

Comment:
Well, it might affect running tests in parallel too. Might be nicer to flow the writer in if the intent is to inspect it rather than declare a static field.